### DISCUSSION TITLE
Can we do `sendTransaction` w/o confirmation from connected wallet.

### DISCUSSION THREAD

**Discussion by ZeeshanAhmadKhalil** [wevm/wagmi#1479] [Q&A]
I have connected my metamask wallet to my web app using `wagmi`. But whenever I am performing `sendTransaction` it is asking for confirmation to my wallet. Can we do it without confirmation?

I want to automate monthly transactions later which will transfer to multiple addresses automatically at the end of the month. But it will not be possible if it keeps asking for confirmation.

**Comment by SaadNiaziDev** [comment]
Automatic transactions or bulk transaction can be acheived by using wallet private key. I havent tried this yet but i guess this is the way to do this

**Reply by ZeeshanAhmadKhalil** [reply]
Is there any hook in wagmi for this?

**Reply by jxom** [reply]
I made a custom `EOAConnector`, which you can attach your private key onto, which may work.

Check out the example: https://github.com/jxom/wagmi-eoa
